FightPandemics/Features/Post/CreatePost/OfferRequest/OfferRequest.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  OfferRequest.swift
+//  FightPandemics
+//
+//  Created by ehsan sat on 5/16/20.
+//
+//  Copyright (c) 2020 FightPandemics
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import UIKit
+
+class OfferRequest: UIView {
+    private var topLine = UIView()
+    private var questionLbl = UILabel()
+    private var offerBtn = UIButton()
+    private var reqBtn = UIButton()
+    private var closeBtn = UIButton()
+    private var confirmBtn = UIButton()
+    private var firstVSV = UIStackView()
+    private var firstHSV = UIStackView()
```

Comment:
The stackViews **firstHSV** and **firstVSV** were not used. It would have been great if you used the stackview for grouping the **close** and **confirm** buttons and just set one constraint for the stackview.